src/components/experiment/ExperimentFilterBar.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { DateParam, NumberParam, useQueryParams } from 'use-query-params';
+
+import CallFilter from 'components/common/proposalFilters/CallFilter';
+import InstrumentFilter from 'components/common/proposalFilters/InstrumentFilter';
+import { DefaultQueryParams } from 'components/common/SuperMaterialTable';
+import { useCallsData } from 'hooks/call/useCallsData';
+import { useInstrumentsData } from 'hooks/instrument/useInstrumentsData';
+
+import DateFilter from './DateFilter';
+import { ExperimentUrlQueryParamsType } from './ExperimentUrlQueryParamsType';
+
+function ExperimentFilterBar() {
+  const [urlQueryParams, setUrlQueryParams] =
+    useQueryParams<ExperimentUrlQueryParamsType>({
+      ...DefaultQueryParams,
+      call: NumberParam,
+      instrument: NumberParam,
+      from: DateParam,
+      to: DateParam,
+    });
+
+  const { instruments, loadingInstruments } = useInstrumentsData();
+  const { calls, loadingCalls } = useCallsData();
+
+  return (
+    <>
+      <CallFilter
+        callId={urlQueryParams.call ?? undefined}
```

Comment:
Maybe it is better option to use `withDefault(NumberParam, undefined)` when defining call inside the `useQueryParams` instead of this check here.

Reply:
I think this is fine. It can be null if users want's to get all results, or undefined if he is not setting anything. If using with default option, I need to exclude the `null` from `ExperimentUrlQueryParamsType['call']` type. This also makes it look less consistent with other parameters.